internal/collector/jaeger/receiver.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package jaegerreceiver wraps the functionality to start the end-point that
+// receives data directly in the Jaeger format as jaeger-collector (UDP as
```

Comment:
@pjanotti should we be supporting that UDP-based Jaeger agent? I ask because when I took a look at the Jaeger docs https://jaeger.readthedocs.io/en/stable/deployment/ the collector seemed like the most encompassing.

Reply:
Comment seems inaccurate, when you are actually supporting the jaeger-collector protocols (tchannel and thrift) and not the jaeger-agent udp format

Reply:
The collector is really the most encompassing but the UDP is intended to be used as a sidecar (not over network), so I think that downline we want to have that but the demand/priority is not clear at this moment.

Reply:
Side note: Jaeger folks consider readthedocs retired we should be using jaegertracing.io (tell that to the search engines :smile:)